en/chromebook_setup/instructions.md

```diff
@@ -2,26 +2,44 @@ You can [skip right over this section](http://tutorial.djangogirls.org/en/instal
 are, your installation experience will be a little different. You can ignore the
 rest of the installation instructions.
 
-### Cloud 9
+### Cloud IDE (PaizaCloud Cloud IDE, AWS Cloud9)
 
-Cloud 9 is a tool that gives you a code editor and access to a computer running
+Cloud IDE is a tool that gives you a code editor and access to a computer running
 on the Internet where you can install, write, and run the software. For the duration
-of the tutorial, Cloud 9 will act as your _local machine_. You'll still be
+of the tutorial, Cloud IDE will act as your _local machine_. You'll still be
```

Comment:
If "Cloud IDE" is not a brand name or proper name, "cloud" probably shouldn be capitalized: Make it "cloud IDE".